express/scripts/scripts.js

```diff
@@ -2381,7 +2381,7 @@ function removeMetadata() {
 
 export async function buildStaticFreePlanWidget() {
   const placeholders = await fetchPlaceholders();
-  const widget = createTag('div', { class: 'free-plan-widget' });
+  const widget = createTag('div', { class: 'free-plan-widget', style: `display: block !important` });
```

Comment:
Can you check if we can avoid using !important here? I actually believe we can debug this from CSS without injecting inline styling.